src/game_server/command.rs

```diff
@@ -41,6 +42,19 @@ impl SerializePacket for CommandOpCode {
     }
 }
 
+#[derive(SerializePacket, DeserializePacket)]
+pub struct ChatBubbleColor {
+    text_color: u32,
+    bubble_color: u32,
```

Comment:
Overall LGTM and I'm fine to approve this as-is. One idea I had is to create a `Rgba` struct that implements `SerializePacket` and `DeserializePacket` so we don't need to  the bit shifting math for these colors ourselves. WDYT?